usaspending_api/awards/models.py

```diff
@@ -310,6 +328,24 @@ def get_default_fields(path=None):
             "assistance_data"  # must match related_name in TransactionAssistance
         ]
 
+    @classmethod
+    def get_or_create(cls, **kwargs):
```

Comment:
Outing myself as on the noob-ier side of Django here, but is it correct to assume that this overrides the model's `get_or_create` function? Would someone still be able to use the regular `get_or_create` with the `defaults` parameter and whatnot?